service-base/src/test/java/org/eclipse/hono/service/auth/EventBusAuthenticationServiceTest.java

```diff
@@ -46,8 +45,8 @@ public class EventBusAuthenticationServiceTest {
     private static Vertx vertx;
 
     private static String secret = "dafhkjsdahfuksahuioahgfqgpsgjkhfdjkg";
-    private static AuthTokenHelper authTokenHelperForSigning;
-    private static AuthTokenHelper authTokenHelperForValidating;
+    private static AuthTokenFactory authTokenHelperForSigning;
+    private static AuthTokenValidator authTokenHelperForValidating;
```

Comment:
The term `helper` could also be replaced in variable names here.

Reply:
Is this all?